sites/svelte.dev/vite.config.js

```diff
@@ -37,6 +37,20 @@ function raw(ext) {
 /** @type {import('vite').UserConfig} */
 const config = {
 	logLevel: 'info',
+	css: {
+		transformer: 'lightningcss',
+		lightningcss: {
+			targets: {
```

Comment:
any reason these particular targets were chosen?

Reply:
Copilot 😅

Reply:
it looks like you can also set something like `'>= 0.25%'`, which may make more sense. https://lightningcss.dev/docs.html

Reply:
Done